CHANGES.rst

```diff
@@ -1,16 +1,20 @@
 Changes
 =======
 
-Version 0.7.3 (UNRELEASED)
+Version 0.7.3 (2021-03-17)
 --------------------------
 
+- Users:
+    - Fixes problem with jobs stuck in `running` state when there are network issues, improving the overall platform resilience.
+    - Fixes problem related to Slurm job submissions.
 - Administrators:
     - Adds new configuration to toggle Kubernetes user jobs clean up.
       (``REANA_RUNTIME_KUBERNETES_KEEP_ALIVE_JOBS_WITH_STATUSES`` in ``components.reana_workflow_controller.environment``)
     - Improves platform resilience.
 - Developers:
     - Adds new flags to ``reana-dev run-example`` allowing full paralell test matrix execution
```

Comment:
Adds new command-line options to ``reana-dev run-example`` command allowing full parallel asynchronous execution of demo examples.